index.js

```diff
@@ -1 +1,109 @@
-// implement your API here
+const express = require("express");
+
+const db = require("./data/db.js");
+const server = express();
+
+// MiddleWare
+server.use(express.json());
+
+// Create - Adds a new user to the db - Endpoint 'api/users'
+server.post("/api/users", (req, res) => {
+  const { name, bio } = req.body;
+  if (!name || !bio) {
```

Comment:
great use of validation and breaking up what you're getting from the req.body object.